cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -4,13 +4,19 @@ kind: Template
 metadata:
   name: cluster-service-admin
 parameters:
-- name: NAMESPACE
-  description: The namespace to create
-  required: true
-  value: cluster-service-admin
-- name: CLIENT_ID
-  description: The Azure Client ID used for federation
-  required: true
+  - name: NAMESPACE
+    description: The namespace to create
+    required: true
+    value: cluster-service-admin
+  - name: CLIENT_ID
+    description: The Azure Client ID used for federation
+    required: true
+  - name: ORPHANED_NAMESPACE_CLEANER
+    description: The namespace to create to have a cronjob which will delete the orphaned namespace which are not deleted due to any issues with the jenkins job.
+    value: orphaned-namespace-cleaner
+  - name: KUBECTL_IMAGE
+    description: An image which have the `kubectl` binary in it.
+    value: quay.io/rhn_support_ansverma/ubi8-minimal-kubectl:latest
```

Comment:
Is there some reason we are referencing a repository of a specific developer for the container image?

Reply:
We can push this image to ACR as well once we are ready for that, till then we can use this one.